src/gravel/controllers/nodes/systemdisk.py

```diff
@@ -73,7 +73,9 @@ def get_mounts() -> List[MountEntry]:
             fields: List[str] = line.split(" ")
             if len(fields) < 2:
                 continue
-            lst.append(MountEntry(source=fields[0], dest=fields[1]))
+            src = fields[0].strip()
+            dst = fields[1].strip()
```

Comment:
(nit) normalize using the Path module? or maybe use `psutil` instead of reading `/proc/mounts`?

Reply:
oh, that's a nice one. Let's add that with the networking changes and then refactor this bit then. I think that will be useful in the systemdisk code in general, not just here, and doing a major refactor with this PR would seem weird :)